lib/proxy-rewrite-cookies.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var serialize = require('cookie').serialize;
+var parse = require('set-cookie-parser').parse;
```

Comment:
I was wondering why we don't use the `parse` function of the `cookie` module but found the answer here: https://github.com/jshttp/cookie/issues/58.

Maybe we could add a comment clarifying that `cookie` can only parse the `Cookie` header and not `Set-Cookie`, which is why we use the other module?